### PR TITLE
Bookstore: Styling

### PR DESCRIPTION
**Pull request by Kumilachew-g**
I have done the following tasks

- I have used the information detailed in [Zeplin](https://zeplin.io/) to style the website. 
- I have done all the details of the Bookstore design looks like  [Zeplin](https://zeplin.io/).
- I have done all book UI pages.
- I have done all book page stylings.
- My final design matched [Zeplin's design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md).

### CODE REVIEW

File: src/components/InputBook.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const InputBook = () => {
 
   return (
     <form onSubmit={handleSubmit}>
```
Comment (julie-ify) [inline src/components/InputBook.js:36]: - [ ] I encounter API-related errors when trying to add a book. Please ensure that your API works as expected.

  neww

File: src/components/Book.js [hunk 2]
```diff
@@ -3,14 +3,47 @@ import PropTypes from 'prop-types';
 import RemoveBook from './RemoveBook';
 
 const Book = ({ id, title, category }) => (
```
Comment (julie-ify) [inline src/components/Book.js:5]: - [ ] Your books from the API are not really fetched so I am unable to check the styling of the book components based on the wireframe. Kindly ensure your API still works as expected

  ndnd

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Kumilachew-g,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kumilachew-g** [comment]
**The UI looks like this:**
image

**Review by fobadara — APPROVED** [review]
Hi @Kumilachew-g, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
✅ Nice commits
✅ Good PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**